tests/test_sidecar2cdxj.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+import surt
+from unittest.mock import patch
+from warcio.timeutils import iso_date_to_timestamp
+
+import sidecar2cdxj
+
+
+TEST_DIR = os.path.dirname(__file__)
+
+TEXT_META_FILE = os.path.join(TEST_DIR, 'text.warc.meta.gz')
+
+WARC_PAYLOAD = json.dumps({'Identified-Payload-Type': {'fido': 'text/html',
```

Comment:
I think it is a little confusing to call this `WARC_PAYLOAD` because it is not something that is a WARC payload, it is the JSON that is expected to appear in the CDXJ that was constructed from the fields that appeared in the WARC's payload.

Reply:
What about SIDECAR_JSON? or CDXJ_JSON?

Reply:
Yeah maybe `CDXJ_JSON`.